go/test/endtoend/vtgate/queries/aggregation/aggregation_test.go

```diff
@@ -116,12 +116,16 @@ func TestGroupBy(t *testing.T) {
 	mcmp.Exec("insert into t3(id5, id6, id7) values(1,1,2), (2,2,4), (3,2,4), (4,1,2), (5,1,2), (6,3,6)")
 	// test ordering and group by int column
 	mcmp.AssertMatches("select id6, id7, count(*) k from t3 group by id6, id7 order by k", `[[INT64(3) INT64(6) INT64(1)] [INT64(2) INT64(4) INT64(2)] [INT64(1) INT64(2) INT64(3)]]`)
-	mcmp.AssertMatches("select id6+id7, count(*) k from t3 group by id6+id7 order by k", `[[INT64(9) INT64(1)] [INT64(6) INT64(2)] [INT64(3) INT64(3)]]`)
+	if utils.BinaryIsAtLeastAtVersion(18, "vtgate") {
+		mcmp.AssertMatches("select id6+id7, count(*) k from t3 group by id6+id7 order by k", `[[INT64(9) INT64(1)] [INT64(6) INT64(2)] [INT64(3) INT64(3)]]`)
+	}
```

Comment:
This also makes sense, support for it was added in v17.0 (https://github.com/vitessio/vitess/commit/bedcf510f1876e781445228a52f90a16891a5503#diff-807c83ede43d6ba771029fc11ac076d6f9c705430df37df98a56205b6eeb81b1) via https://github.com/vitessio/vitess/pull/12994